packages/fields-document/src/index.ts

```diff
@@ -7,6 +7,7 @@ import {
   graphql,
   JSONValue,
 } from '@keystone-next/keystone/types';
+import { userInputError } from '@keystone-next/keystone/src/lib/core/graphql-errors';
```

Comment:
This will break when shipped to npm so it needs to be exported publically and when you do so, please make it a class